controllers/storagecluster/provider_server.go

```diff
@@ -454,6 +454,7 @@ func getOnboardingJobObject(instance *ocsv1.StorageCluster) *batchv1.Job {
 			Namespace: instance.Namespace,
 		},
 		Spec: batchv1.JobSpec{
+			TTLSecondsAfterFinished: util.ToPointer[int32](0), //Eligible to delete automatically when job completes
```

Comment:
Isn't `util.ToPointer(int32(0))` more safer, like ide may throw error w/ `bool(0)` but not for `[bool](0)` 🤔. I believe compiler throws error for both ways though. Pls check, can stay as it is if it's not creating any issue.

Reply:
i have checked `[bool](0)` compiler is not throwing error for this. We can stay with this